requirements.txt

```diff
@@ -1,156 +1,201 @@
-appdirs==1.4.0
-astroid==1.4.8
-Babel==2.3.4
-backports.shutil-get-terminal-size==1.0.0
-beautifulsoup4==4.5.1
-boto3==1.9.64
-botocore==1.12.64
-chardet==2.3.0
-CherryPy==3.8.2
-click==6.6
-cliff==2.3.0
-cmd2==0.6.9
-colorama==0.3.7
+ansible==2.3.2.0
+appdirs==1.4.3
+asn1crypto==0.24.0
+astroid==2.1.0
+attrs==19.1.0
+Babel==2.7.0
+backcall==0.1.0
+backports.functools-lru-cache==1.5
+bcrypt==3.1.4
+beautifulsoup4==4.7.1
+boto==2.48.0
+boto3==1.9.174
+botocore==1.12.174
+certifi==2019.6.16
+cffi==1.12.3
+chardet==3.0.4
+cheroot==6.5.5
+CherryPy==18.1.1
+Click==7.0
+cliff==2.15.0
+cmd2==0.9.13
+colorama==0.4.1
 colour==0.1.5
 cookies==2.2.1
-coverage==4.2
-ddt==1.1.1
-debtcollector==1.9.0
-decorator==4.0.10
-Django==1.11.20
--e git+https://github.com/jrief/django-angular.git@3a4eb9e716b0f03e4abe5ec60d6978a52da100f8#egg=django_angular
-django-appconf==1.0.2
-django-compressor==2.1
-django-debug-toolbar==1.6
-django-environ==0.4.0
-django-extensions==1.7.4
-django-filter==0.15.3
-django-grappelli==2.8.3
+coverage==4.5.3
+cryptography==2.7
+ddt==1.2.1
+debtcollector==1.21.0
+decorator==4.4.0
+Django==2.2.2
+# with Django 1 we were using: -e git+https://github.com/jrief/django-angular.git@3a4eb9e716b0f03e4abe5ec60d6978a52da100f8#egg=django_angular	
+# with Django 2, clean option:
+# django-angular==2.2
+# but django-angular 2.2 requires a heavy rewrite of many parts of the form. Disabling django-angular does too. So a compromise is to patch our django-angular version to work with Django 2:
+# TODO: use opencraft's branch
+-e git+https://github.com/open-craft/django-angular.git@clemente/migrate-our-django-angular-to-django-2#egg=django_angular
```

Comment:
This TODO must be resolved before merging

Reply:
I fixed it and I really moved it to a branch without my username, https://github.com/open-craft/django-angular/tree/v1-with-django2-support